File: abc/CharacterGenerator/Lambda.js
"use strict"

if (module === require.main) {
    const FIRST_GLYPH = 33 // '!'
    const LAST_GLYPH = 126 // '~'
    const GLYPHS_PER_LINE = 72

    let cursor = FIRST_GLYPH

    let i = 0
    while (true) {
        i = cursor
        for (let j = 0; j < GLYPHS_PER_LINE; j++) {
            process.stdout.write(String.fromCharCode(i))
            if (i === LAST_GLYPH) {
                i = FIRST_GLYPH
            } else {
                i += 1
            }
        }
        console.log()
        if (cursor === LAST_GLYPH) {
            cursor = FIRST_GLYPH
        } else {
            cursor += 1
        }
    }
}

File: abc/CharacterGenerator/Lambda.py
def main() -> None:
    FIRST_GLYPH = 33 # '!'
    LAST_GLYPH = 126 # '~'
    GLYPHS_PER_LINE = 72

    cursor = FIRST_GLYPH

    i = 0
    while True:
        i = cursor
        for _ in range(GLYPHS_PER_LINE + 1):
            print(chr(i), end = "")
            if i == LAST_GLYPH:
                i = FIRST_GLYPH
            else:
                i += 1
        print()
        if cursor == LAST_GLYPH:
            cursor = FIRST_GLYPH
        else:
            cursor += 1

if __name__ == "__main__":
    main()
